dot/state/epoch.go

```diff
@@ -512,6 +512,30 @@ func (s *EpochState) SkipVerify(header *types.Header) (bool, error) {
 	return false, nil
 }
 
+// AlreadyDefined will true for each returned value if it is already defined in the database for the given epoch
+func (s *EpochState) AlreadyDefined(epoch uint64) (epochData bool, configData bool, err error) {
+	keys := [...][]byte{epochDataKey(epoch), configDataKey(epoch)}
+	applied := [...]bool{false, false}
```

Comment:
Why use arrays? Can't we use some `epochDataKeyByte` and `configDataKeyByte` variables instead, and update `epochData` and `configData` without a for loop?

That would be a bit longer code, but quite easier to follow/read I think.

Also I would think splitting this function in two, one for epochData and one for configData would be nice (without using generics lol).